client/src/Components/Card/CarNameCard.js

```diff
@@ -1,27 +1,16 @@
-import React, { Component } from "react";
```

Comment:
likewise

Reply:
Eventually, I want to display cars based on how many cars the particular user owns. 
Commented out for the time being.